### DISCUSSION TITLE
Assertion errors "fix"

### DISCUSSION THREAD

**Discussion by jgserra18** [aquacropos/aquacrop#69] [Ideas]
The assertion errors related to crop maturity and the gdd.sum make it impossible to run aquacrop in a grid. This is particularly systematic at a relatively fine resolution. I was thinking it would be a good idea to add an option (e.g., a boolean grid_simulation argument)  that would disable the assertion errors. Returning 0 or -999 would be a relatively easy option to stop the model to further simulate in such cases, albeit not quite very elegant (e.g., in compute_crop_calendar --> compute_variables --> initialize --> core.

**Comment by jgserra18** [comment]
Never mind me, this is easy to fix with a try and except.

**Comment by thomasdkelly** [comment]
Hi @jgserra18, Yes this is something I have thought about (i.e., allowing a simulation to contintue even though the GDD's are not enough to reach full maturity), just havent got around to it yet. I believe it did cause problems with other calculations which is which I introuduced the assertion but it has been a while since I looked at that aspect. Feel free to have a go a removing it in a pull request or go ahead with your try/except fix. 

Tom
